packages/cli-core/API.md

```diff
@@ -29,6 +29,14 @@ export enum COLOR {
 
 // @public
 export const format: {
+    runner: (binaryRunner: string) => {
+        amplifyCommand: (command: string) => string;
+    };
+    infoMessage: (message: string) => string;

```

Comment:
`infoMessage` could be confused with an info log line. I think I prefer my original suggestion of `note` or maybe `aside` if we're trying to convey that it's not a super important message